docs/LSPSupport.md

```diff
@@ -458,6 +458,67 @@ Here is an example with the [Java Language Server](https://github.com/eclipse-jd
 
 ![textDocument/onTypeFormatting](./images/lsp-support/textDocument_onTypeFormatting.gif)
 
+If desired &mdash; for example, for those who want to control when formatting is performed &mdash; server-side/LSP-based

```

Comment:
While updating the on-type formatting docs for this feature, I added a section for how to disable server-side on-type formatting if desired.